src/repl/scala/tools/nsc/interpreter/InteractiveReader.scala

```diff
@@ -46,6 +46,13 @@ object InteractiveReader {
     }
 
   def apply(): InteractiveReader = SimpleReader()
+
+  // a non-interactive SimpleReader that returns the given text
+  def apply(text: String): InteractiveReader = SimpleReader(
```

Comment:
Yes, I expected a clean break from spark support at some point. Also, Ammonite is asking for community funding. What a great idea. I mean dollars, not just beer support.